tools/ci_build/github/azure-pipelines/linux-ci-pipeline-with-java.yml

```diff
@@ -0,0 +1,9 @@
+jobs:
+- template: templates/linux-ci.yml
+  parameters:
+    AgentPool : 'Linux-CPU'
+    JobName: 'Linux_CI_Dev'
+    BuildCommand: 'tools/ci_build/github/linux/run_dockerbuild.sh -o ubuntu16.04 -d cpu -r $(Build.BinariesDirectory) -x "--use_mklml --use_llvm --use_nuphar --use_mkldnn --use_tvm --use_automl --build_wheel --build_java --enable_language_interop_ops"'
```

Comment:
Can we combine this pipeline with Linux CPU? It will help us save some resource.

Reply:
Yes, that’s the plan after the Java PR is checked in. Otherwise —build_java will fail the builds